_posts/2020-02-07-metal3-dev-env-install-deep-dive.md

```diff
@@ -0,0 +1,921 @@
+REPOSITORY                                TAG                 IMAGE ID            CREATED             SIZE
+quay.io/metal3-io/ironic                  latest              e5d81adf05ee        26 hours ago        693MB
+quay.io/metal3-io/ironic-ipa-downloader   latest              d55b0dac2144        6 days ago          239MB
+quay.io/metal3-io/ironic-inspector        latest              8bb5b844ada6        6 days ago          408MB
+quay.io/metal3-io/baremetal-operator      latest              3c692a32ddd6        9 days ago          1.77GB
+k8s.gcr.io/kube-proxy                     v1.17.0             7d54289267dc        7 weeks ago         116MB
+k8s.gcr.io/kube-controller-manager        v1.17.0             5eb3b7486872        7 weeks ago         161MB
+k8s.gcr.io/kube-scheduler                 v1.17.0             78c190f736b1        7 weeks ago         94.4MB
+k8s.gcr.io/kube-apiserver                 v1.17.0             0cae8d5cc64c        7 weeks ago         171MB
+kubernetesui/dashboard                    v2.0.0-beta8        eb51a3597525        7 weeks ago         90.8MB
+k8s.gcr.io/coredns                        1.6.5               70f311871ae1        2 months ago        41.6MB
+k8s.gcr.io/etcd                           3.4.3-0             303ce5db0e90        3 months ago        288MB
+kubernetesui/metrics-scraper              v1.0.2              3b08661dc379        3 months ago        40.1MB
+k8s.gcr.io/kube-addon-manager             v9.0.2              bd12a212f9dc        6 months ago        83.1MB
+k8s.gcr.io/pause                          3.1                 da86e6ba6ca1        2 years ago         742kB
+gcr.io/k8s-minikube/storage-provisioner   v1.8.1              4689081edb10        2 years ago         80.8MB
+```
+
+Once container images are stored, `minikube` can be stopped. In that moment, the virtual networks shown in the previous picture are attached to the `minikube` VM as can be verified by the following command: 
+
+```sh
+[alosadag@smc-master metal3-dev-env]$ sudo virsh domiflist minikube
+Interface  Type       Source     Model       MAC
+-------------------------------------------------------
+-          network    default    virtio      d4:38:25:25:c6:ca
+-          network    minikube-net virtio      a4:c2:8a:9d:2a:d8
+-          network    provisioning virtio      52:54:00:c8:50:97
+-          network    baremetal  virtio      52:54:00:17:b4:ec
+```
+
+> **Information:** At this point the host is ready to create the virtual infrastucture.
+
+<br>
+
+<iframe width="1110" height="625" style="height: 625px" src="https://www.youtube.com/embed/lShd0RxDhFQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<br>
+
+## **Step 2: Configure the host**
+
+
+In this step, `02_configure_host.sh` basically configures the libvirt/KVM virtual infrastructure and starts services in the host that will be consumed by the virtual bare metal machines: 
```

Comment:
In this step, the script `02_configure_host.sh` basically configures the libvirt/KVM...